addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1054,6 +1077,22 @@ const actions = {
             chatWindowInitialScrollTops: newChatWindowInitialScrollTops,
         });
     },
+    /**
+     * @param {Object} state
+     * @param {string} composerLocalId
+     * @param {Object} data
+     */
+    saveStateInStore(
```

Comment:
rename to `saveComposerTextInputInStore`

Reply:
Done